content/en/Integrations/_index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Cobalt Integrations"
+linkTitle: "Integrations"
+weight: 110
+description: >
+  Integrate third-party apps and configure webhooks.
+---
+
+{{% pageinfo %}}
+Streamline your pentesting and development workflows with Cobalt integrations.
+{{% /pageinfo %}}
+
+{{% alert title="Note" color="note" %}}
+The availability of this feature is based on your <a href="https://www.cobalt.io/pentest-pricing" target="_blank">PtaaS tier</a>.
+{{% /alert %}}
+ 
+In the Cobalt app, navigate to the **Integrations** page to get started.
```

Comment:
```suggestion
In the Cobalt app, navigate to the **Integrations** page to get started.
```

I'm assuming that the user has already signed in and is now exploring the app functionality. We only have one app, so I think it's OK to mention "in the Cobalt app." Please let me know if it works. Maybe it's my habit from my previous projects.